ingestion/src/metadata/utils/sqlalchemy_utils.py

```diff
@@ -125,3 +126,26 @@ def convert_numpy_to_list(data):
     if isinstance(data, dict):
         return {key: convert_numpy_to_list(value) for key, value in data.items()}
     return data
+
+
+@reflection.cache
+def get_all_table_ddls(
+    self, connection, query, schema_name, **kw
+):  # pylint: disable=unused-argument
+    """
+    Method to fetch ddl of all available tables
+    """
+    self.all_table_ddls: Dict[Tuple[str, str], str] = {}
```

Comment:
Might be better to add some exception handling here

Reply:
Added exception here